frontend/src/App.tsx

```diff
@@ -111,6 +111,7 @@ const ELEMENT_LIST_BUFFER_TIMEOUT_MS = 10
 declare global {
   interface Window {
     streamlitDebug: any
+    streamlitTracking: Record<string, unknown>
```

Comment:
Can you rename this to something clearer? Like `streamlitShareMetadata` or something?

Because "Streamlit tracking" could mean all sorts of things...

Reply:
Renamed to `streamlitShareMetadata`